packages/base-styles/_z-index.scss

```diff
@@ -155,8 +155,7 @@ $z-layers: (
 	".components-popover.customize-widgets-more-menu__content": 99998,
 	".components-popover.edit-post-more-menu__content": 99998,
 	".components-popover.edit-site-more-menu__content": 99998,
-	".components-popover.edit-widgets-more-menu__content": 99998,
-	".components-popover.preferences-more-menu__content": 99998,
```

Comment:
Seems like I accidentally added this in https://github.com/WordPress/gutenberg/pull/38873. The more menu is staying in the interface package, so the 'preferences' part here is wrong.

It shouldn't have caused any bugs as nothing should be using this name. As I migrate each editor, the individual z-indices for each editor can be removed in favor of just the `interface` one.